src/app/clusters/occupancy-sensor-server/occupancy-sensor-server.cpp

```diff
@@ -59,7 +59,10 @@ CHIP_ERROR Instance::Read(const ConcreteReadAttributePath & aPath, AttributeValu
     case Attributes::FeatureMap::Id:
         ReturnErrorOnFailure(aEncoder.Encode(mFeature));
         break;
-    case Attributes::HoldTime::Id: {
+    case Attributes::HoldTime::Id:
+    case Attributes::PIROccupiedToUnoccupiedDelay::Id:
+    case Attributes::UltrasonicOccupiedToUnoccupiedDelay::Id:
+    case Attributes::PhysicalContactOccupiedToUnoccupiedDelay::Id: {
```

Comment:
So these other attributes are expected to always be the same as HoldTime even "out of the box" before anyone has done a write to HoldTime?  I don't think the old setup forced that...

It might be worth documenting the exact setup here.

Reply:
Yes, the SPEC says `HoldTime is equivalent to the legacy *OccupiedToUnoccupiedDelay attributes.`. And in the old setup when an *OccupiedToUnoccupiedDelay attribute is change by a write operation, the HoldTime and other *OccupiedToUnoccupiedDelay attributes will also been changed.

Added the document.